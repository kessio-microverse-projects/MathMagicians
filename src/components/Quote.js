import React from 'react';
import style from './Home.module.css';

export default function Quote() {
  return (
    <div className={style.quoteSection}>
      Mathematics is not about numbers, equations, computations, or algorithms.
      It is about understanding. -William Paul Thurston
    </div>
  );
}
