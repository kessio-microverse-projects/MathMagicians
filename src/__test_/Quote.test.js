import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('snapshot test for Quote Component', () => {
  it('Matches the snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot('Quote');
  });
});
