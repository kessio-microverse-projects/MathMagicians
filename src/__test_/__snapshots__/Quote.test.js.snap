// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for Quote Component Matches the snapshot: Quote 1`] = `
<div
  className="quoteSection"
>
  Mathematics is not about numbers, equations, computations, or algorithms. It is about understanding. -William Paul Thurston
</div>
`;
